todo_app/templates/form.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'layout.html' %}
+
+{% block content %}
+  <h4 class="title">New Task</h2>
+    <form action="{{ url_for('add') }}" method="POST">
```

Comment:
You actually want `url_for('post_item')` (though this works because currently they are the same URL)